GNOME updates 2025-02-08

jtojnar opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Skimmed through upstream changes
- [x] Read upstream news
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

jtojnar committed:
epiphany: 47.2 → 47.3.1

https://gitlab.gnome.org/GNOME/epiphany/-/compare/47.2...47.3.1

jtojnar committed:
gcr_4: 4.3.0 → 4.3.1

https://gitlab.gnome.org/GNOME/gcr/-/compare/4.3.0...4.3.1

jtojnar committed:
ghex: 46.1 → 46.2

https://gitlab.gnome.org/GNOME/ghex/-/compare/46.1...46.2

jtojnar committed:
gnome-connections: 47.0 → 47.2.1

https://gitlab.gnome.org/GNOME/gnome-connections/-/compare/47.0...47.2.1

jtojnar committed:
gnome-control-center: 47.3 → 47.4

https://gitlab.gnome.org/GNOME/gnome-control-center/-/compare/47.3...47.4

jtojnar committed:
gnome-initial-setup: 47.2 → 47.4

https://gitlab.gnome.org/GNOME/gnome-initial-setup/-/compare/47.2...47.4

jtojnar committed:
gnome-mahjongg: 47.0 → 47.1

https://gitlab.gnome.org/GNOME/gnome-mahjongg/-/compare/47.0...47.1

jtojnar committed:
gnome-maps: 47.3 → 47.4

https://gitlab.gnome.org/GNOME/gnome-maps/-/compare/47.3...47.4

jtojnar committed:
gnome-shell: 47.3 → 47.4

https://gitlab.gnome.org/GNOME/gnome-shell/-/compare/47.3...47.4

jtojnar committed:
gnome-system-monitor: 47.0 → 47.1

https://gitlab.gnome.org/GNOME/gnome-system-monitor/-/compare/47.0...47.1

jtojnar committed:
gnome-text-editor: 47.2 → 47.3

https://gitlab.gnome.org/GNOME/gnome-text-editor/-/compare/47.2...47.3

jtojnar committed:
gtk-vnc: 1.4.0 → 1.5.0

https://gitlab.gnome.org/GNOME/gtk-vnc/-/compare/1.4.0...1.5.0

jtojnar committed:
libshumate: 1.3.1 → 1.3.2

https://gitlab.gnome.org/GNOME/libshumate/-/compare/1.3.1...1.3.2

jtojnar committed:
nautilus: 47.1 → 47.2

https://gitlab.gnome.org/GNOME/nautilus/-/compare/47.1...47.2

jtojnar committed:
mutter: 47.4 → 47.5

https://gitlab.gnome.org/GNOME/mutter/-/compare/47.4...47.5

jtojnar committed:
zenity.updateScript: Fix version policy

Zenity 4.1.90 is the first alpha release of Zenity for the 4.2 release cycle:
https://gitlab.gnome.org/GNOME/zenity/-/blob/4.1.90/NEWS

jtojnar committed:
libpanel: Fix version policy

1.9.0 is a beta release for GNOME 48.beta:
https://gitlab.gnome.org/GNOME/libpanel/-/blob/1.9.0/NEWS

bjornfor reviewed (approved):
I read all commits.

philiptaron reviewed (approved):
`.nix` changes look good, doing a build.

philiptaron commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 380474`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>chatty</li>
    <li>mailnagWithPlugins</li>
    <li>mailnagWithPlugins.dist</li>
    <li>xmonad_log_applet</li>
  </ul>
</details>
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>streamcontroller</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 236 packages built:</summary>
  <ul>
    <li>adapta-gtk-theme</li>
    <li>almanah</li>
    <li>appvm</li>
    <li>arc-theme</li>
    <li>ashpd-demo</li>
    <li>ayatana-indicator-datetime</li>
    <li>ayatana-indicator-session</li>
    <li>ayu-theme-gtk</li>
    <li>bubblemail</li>
    <li>bubblemail.dist</li>
    <li>budgie-analogue-clock-applet</li>
    <li>budgie-control-center</li>
    <li>budgie-control-center.debug</li>
    <li>budgie-desktop</li>
    <li>budgie-desktop-with-plugins</li>
    <li>budgie-desktop.dev</li>
    <li>budgie-desktop.man</li>
    <li>budgie-gsettings-overrides</li>
    <li>budgie-session</li>
    <li>budgie-session.debug</li>
    <li>budgie-session.man</li>
    <li>budgie-user-indicator-redux</li>
    <li>calls</li>
    <li>calls.devdoc</li>
    <li>cheese</li>
    <li>cheese.devdoc</li>
    <li>cheese.man</li>
    <li>cinnamon-common</li>
    <li>cinnamon-control-center</li>
    <li>cinnamon-gsettings-overrides</li>
    <li>cinnamon-screensaver</li>
    <li>collision</li>
    <li>devtoolbox</li>
    <li>dropbox-cli</li>
    <li>dropbox-cli.nautilusExtension</li>
    <li>eiciel</li>
    <li>eiciel.nautilusExtension</li>
    <li>endeavour</li>
    <li>eolie</li>
    <li>epiphany</li>
    <li>equilux-theme</li>
    <li>evolution</li>
    <li>evolution-data-server</li>
    <li>evolution-data-server-gtk4</li>
    <li>evolution-data-server-gtk4.dev</li>
    <li>evolution-data-server.dev</li>
    <li>evolution-ews</li>
    <li>evolutionWithPlugins</li>
    <li>file-roller</li>
    <li>firewalld-gui</li>
    <li>folks</li>
    <li>folks.dev</li>
    <li>folks.devdoc</li>
    <li>fractal</li>
    <li>gcr_4</li>
    <li>gcr_4.bin</li>
    <li>gcr_4.dev</li>
    <li>gcr_4.devdoc</li>
    <li>geary</li>
    <li>gfbgraph</li>
    <li>gfbgraph.dev</li>
    <li>gfbgraph.devdoc</li>
    <li>ghex</li>
    <li>ghex.dev</li>
    <li>ghex.devdoc</li>
    <li>gnome-applets</li>
    <li>gnome-browser-connector</li>
    <li>gnome-calendar</li>
    <li>gnome-connections</li>
    <li>gnome-contacts</li>
    <li>gnome-control-center</li>
    <li>gnome-control-center.debug</li>
    <li>gnome-disk-utility</li>
    <li>gnome-flashback</li>
    <li>gnome-initial-setup</li>
    <li>gnome-mahjongg</li>
    <li>gnome-maps</li>
    <li>gnome-music</li>
    <li>gnome-notes</li>
    <li>gnome-online-accounts</li>
    <li>gnome-online-accounts-gtk</li>
    <li>gnome-online-accounts.debug</li>
    <li>gnome-online-accounts.dev</li>
    <li>gnome-online-accounts.devdoc</li>
    <li>gnome-online-accounts.man</li>
    <li>gnome-panel</li>
    <li>gnome-panel-with-modules</li>
    <li>gnome-panel.dev</li>
    <li>gnome-panel.man</li>
    <li>gnome-photos</li>
    <li>gnome-photos.installedTests</li>
    <li>gnome-recipes</li>
    <li>gnome-session</li>
    <li>gnome-session.debug</li>
    <li>gnome-session.sessions</li>
    <li>gnome-settings-daemon</li>
    <li>gnome-settings-daemon46 (pantheon.gnome-settings-daemon)</li>
    <li>gnome-shell</li>
    <li>gnome-shell.debug</li>
    <li>gnome-shell.devdoc</li>
    <li>gnome-system-monitor</li>
    <li>gnome-terminal</li>
    <li>gnome-text-editor</li>
    <li>gnome-tweaks</li>
    <li>gnome.gvfs</li>
    <li>gnome.gvfs.debug</li>
    <li>gnome.nixos-gsettings-overrides</li>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.gsconnect.installedTests</li>
    <li>gnomeExtensions.gtk4-desktop-icons-ng-ding</li>
    <li>gpaste</li>
    <li>grilo-plugins</li>
    <li>gtk-vnc</li>
    <li>gtk-vnc.bin</li>
    <li>gtk-vnc.dev</li>
    <li>gtk-vnc.devdoc</li>
    <li>gtk-vnc.man</li>
    <li>insync-nautilus</li>
    <li>jogger</li>
    <li>libgdata</li>
    <li>libgdata.dev</li>
    <li>libgdata.installedTests</li>
    <li>libmsgraph</li>
    <li>libmsgraph.dev</li>
    <li>libmsgraph.devdoc</li>
    <li>libnma</li>
    <li>libnma-gtk4</li>
    <li>libnma-gtk4.dev</li>
    <li>libnma-gtk4.devdoc</li>
    <li>libnma.dev</li>
    <li>libnma.devdoc</li>
    <li>libshumate</li>
    <li>libshumate.dev</li>
    <li>libshumate.devdoc</li>
    <li>libzapojit</li>
    <li>libzapojit.dev</li>
    <li>lomiri.lomiri</li>
    <li>lomiri.lomiri-session</li>
    <li>lomiri.lomiri-system-settings</li>
    <li>lomiri.lomiri-system-settings-unwrapped</li>
    <li>lomiri.lomiri-system-settings-unwrapped.dev</li>
    <li>lounge-gtk-theme</li>
    <li>magpie</li>
    <li>magpie.debug</li>
    <li>magpie.dev</li>
    <li>magpie.devdoc</li>
    <li>materia-theme</li>
    <li>materia-theme-transparent</li>
    <li>matrix-gtk-theme</li>
    <li>mojave-gtk-theme</li>
    <li>monitor</li>
    <li>mutter</li>
    <li>mutter.debug</li>
    <li>mutter.dev</li>
    <li>mutter.devdoc</li>
    <li>mutter.man</li>
    <li>mutter46 (pantheon.mutter)</li>
    <li>mutter46.debug (pantheon.mutter.debug)</li>
    <li>mutter46.dev (pantheon.mutter.dev)</li>
    <li>mutter46.devdoc (pantheon.mutter.devdoc)</li>
    <li>mutter46.man (pantheon.mutter.man)</li>
    <li>nautilus</li>
    <li>nautilus-open-any-terminal</li>
    <li>nautilus-open-any-terminal.dist</li>
    <li>nautilus-open-in-blackbox</li>
    <li>nautilus-python</li>
    <li>nautilus-python.dev</li>
    <li>nautilus-python.devdoc</li>
    <li>nautilus-python.doc</li>
    <li>nautilus.dev</li>
    <li>nautilus.devdoc</li>
    <li>nemo-fileroller</li>
    <li>nemo-with-extensions</li>
    <li>networkmanager-fortisslvpn</li>
    <li>networkmanager-iodine</li>
    <li>networkmanager-l2tp</li>
    <li>networkmanager-openconnect</li>
    <li>networkmanager-openvpn</li>
    <li>networkmanager-sstp</li>
    <li>networkmanager-vpnc</li>
    <li>networkmanager_strongswan</li>
    <li>networkmanagerapplet</li>
    <li>networkmanagerapplet.man</li>
    <li>pantheon.elementary-calendar</li>
    <li>pantheon.elementary-capnet-assist</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-gsettings-schemas</li>
    <li>pantheon.elementary-mail</li>
    <li>pantheon.elementary-onboarding</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.elementary-shortcut-overlay</li>
    <li>pantheon.elementary-tasks</li>
    <li>pantheon.epiphany</li>
    <li>pantheon.file-roller-contract</li>
    <li>pantheon.gala</li>
    <li>pantheon.switchboard-plug-bluetooth</li>
    <li>pantheon.switchboard-plug-keyboard</li>
    <li>pantheon.switchboard-plug-mouse-touchpad</li>
    <li>pantheon.switchboard-plug-network</li>
    <li>pantheon.switchboard-plug-onlineaccounts</li>
    <li>pantheon.switchboard-plug-pantheon-shell</li>
    <li>pantheon.switchboard-plug-power</li>
    <li>pantheon.switchboard-plug-security-privacy</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-a11y</li>
    <li>pantheon.wingpanel-indicator-bluetooth</li>
    <li>pantheon.wingpanel-indicator-datetime</li>
    <li>pantheon.wingpanel-indicator-keyboard</li>
    <li>pantheon.wingpanel-indicator-network</li>
    <li>pantheon.wingpanel-indicator-nightlight</li>
    <li>pantheon.wingpanel-indicator-notifications</li>
    <li>pantheon.wingpanel-indicator-power</li>
    <li>pantheon.wingpanel-indicator-sound</li>
    <li>pantheon.wingpanel-quick-settings</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>paper-plane</li>
    <li>papers</li>
    <li>papers.dev</li>
    <li>papers.devdoc</li>
    <li>phoc</li>
    <li>phosh</li>
    <li>phosh-mobile-settings</li>
    <li>planify</li>
    <li>shortwave</li>
    <li>tokyonight-gtk-theme</li>
    <li>totem</li>
    <li>valent</li>
    <li>vimix-gtk-themes</li>
    <li>virt-manager</li>
    <li>virt-viewer</li>
    <li>wingpanel-indicator-ayatana</li>
    <li>wingpanel-indicator-namarupa</li>
    <li>xdg-desktop-portal-gtk</li>
  </ul>
</details>

bobby285271 committed:
gnome-mahjongg: 47.1 → 47.2

https://gitlab.gnome.org/GNOME/gnome-mahjongg/-/compare/47.1...47.2

bobby285271 committed:
gnome-connections: Update meta.homepage

https://discourse.gnome.org/t/announcing-the-rename-of-the-gnome-connections-gitlab-repo-to-gnome-connections/26571

bobby285271 committed:
gtk-frdp: 0-unstable-2024-07-03 → 0-unstable-2024-12-23

https://gitlab.gnome.org/GNOME/gtk-frdp/-/compare/6cfdc840159bb349310c3b81cd2df949f1522760...46ca0beb9b5bf8c9b245a596231016bcca9baf6b
